WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormTableViewDataSource.swift

```diff
@@ -110,53 +109,49 @@ private extension ProductFormTableViewDataSource {
                            productUIImageLoader: productUIImageLoader)
             return
         }
-
         if productImageStatuses.count > 0 {
             if allowsMultipleImages {
-                cell.configure(with: productImageStatuses, config: .addImages, productUIImageLoader: productUIImageLoader)
+                cell.configure(with: productImageStatuses,
+                               config: .addImages,
+                               productUIImageLoader: productUIImageLoader)
             } else {
                 cell.configure(with: productImageStatuses, config: .images, productUIImageLoader: productUIImageLoader)
             }
         }
         else {
             cell.configure(with: productImageStatuses, config: .extendedAddImages, productUIImageLoader: productUIImageLoader)
         }
-
         cell.onImageSelected = { [weak self] (productImage, indexPath) in
             self?.onAddImage?()
         }
         cell.onAddImage = { [weak self] in
             self?.onAddImage?()
         }
     }
-
-    func configureName(cell: UITableViewCell, name: String?, isEditable: Bool) {
+    func configureName(cell: UITableViewCell, name: String?, isEditable: Bool, productLabel: ProductStatus) {
         if isEditable {
-            configureEditableName(cell: cell, name: name)
+            configureEditableName(cell: cell, name: name, productLabel: productLabel)
         } else {
             configureReadonlyName(cell: cell, name: name ?? "")
         }
     }
 
-    func configureEditableName(cell: UITableViewCell, name: String?) {
-        guard let cell = cell as? TextViewTableViewCell else {
+    func configureEditableName(cell: UITableViewCell, name: String?, productLabel: ProductStatus) {
```

Comment:
Minor nitpick, but if we're using "status" both for the data source and the cell view model, maybe we should use `productStatus` here as well.